src/Helper/AbstractHtmlElement.php

```diff
@@ -12,13 +12,6 @@
 abstract class AbstractHtmlElement extends AbstractHelper
 {
     /**
-     * EOL character
-     *
-     * @deprecated just use PHP_EOL
-     */
-    const EOL = PHP_EOL;
```

Comment:
We cannot remove it until a new major version is released.

That said, we can remove usage of it within the tests. When I merge, I'll reinstate the constant.